woocommerce/woocommerce-android
# Hide the red dot from payments button

Closes: #7022


### Description

The PR hides the red badge after a user clicks on it

### Testing instructions

* Go to the More screen
* Notice red badge on the payments icon
* Click on the icon and come back
* Notice that the badge is gone

### Images/gif


https://user-images.githubusercontent.com/4923871/181201342-692a7d5d-a57e-4c31-ab3b-6c2a6e87643a.mp4
